kge/misc.py

```diff
@@ -15,6 +18,33 @@ def is_number(s, number_type):
         return False
 
 
+def merge_dicts_of_1dim_torch_tensors(keys_from_dols, values_from_dols=None):
```

Comment:
these methods can be dropped with the changes suggested above

Reply:
I still need this method for KvsAll, right?